### PR TITLE
fix bad default logic

### PR DESCRIPTION
**Pull request by skylarbarrera**
## What changed (plus any additional context for devs)
fixes bad default logic, no need to migration

## PoW (screenshots / screen recordings)
https://cloud.skylarbarrera.com/Screen-Recording-2022-03-22-11-30-57.mp4

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?

### CODE REVIEW

**Review by jinchung — APPROVED** [review]
🌮

File: src/hooks/usePersistentAspectRatio.ts [hunk 1]
```diff
@@ -44,9 +44,6 @@ export default function usePersistentAspectRatio(url: string): Result {
         },
         () => setState(State.failed)
       );
-    } else {
-      setAspectRatio(1);
-      setState(State.loaded);
     }
   }, [setAspectRatio, state, nonSvgUrl, url]);
```
Comment (brunobar79) [inline src/hooks/usePersistentAspectRatio.ts:51]: url can be removed from here now, right?

File: src/hooks/usePersistentDominantColorFromImage.ts [hunk 2]
```diff
@@ -37,15 +36,13 @@ export default function usePersistentDominantColorFromImage(
...
       setState(State.loading);
       getDominantColorFromImage(lowResUrl, colorToMeasureAgainst).then(color =>
         // @ts-ignore
         setPersistentDominantColor(color)
       );
-    } else {
-      setPersistentDominantColor(colors.paleBlue);
     }
   }, [
     colorToMeasureAgainst,
```
Comment (brunobar79) [inline src/hooks/usePersistentDominantColorFromImage.ts:51]: Same here

**Review by brunobar79 — APPROVED** [review]
LGTM!

**Review by brunobar79 — APPROVED** [review]

**Pull request merged**